src/plugins/data_source/server/routes/fetch_data_source_metadata.test.ts

```diff
@@ -366,6 +373,7 @@ describe(`Fetch DataSource MetaData ${URL}`, () => {
       .expect(200);
     expect(result.body).toEqual({
       dataSourceVersion: '2.11.0',
+      dataSourceEngineType: 'OpenSearch',
       installedPlugins: ['opensearch-ml', 'opensearch-sql'],
```

Comment:
Is this is the only place where we are comparing result.body during testing? What if we are testing result.body at other places like functional, integration? It will break, right? We will need to include dataSourceEngineType everyehwhere.

Reply:
yes as `dataSourceEngineType` is marked as optional to ensure existing test won't fail